Qt.py

```diff
@@ -1800,6 +1800,30 @@ def __call__(self, *a, **kw):
         raise NotImplementedError(self.__err)
 
 
+def _pyqt_enums_patch():
+    """Patch ``PyQt*`` Qt namespace enums to behave like ``PySide*``.
+
+    i.e. ``PyQt*`` only has Qt.Checked and Qt.CheckState, but not ``PySide*``
+    Qt.CheckState.Checked attribute or Qt.CheckState.values dictionary.
+    """
+    qt_attrs = {
+        name: getattr(Qt.QtCore.Qt, name)
+        for name in dir(Qt.QtCore.Qt)
+        if not name.startswith("_")
+    }
```

Comment:
This one for example could (and likely is) pick up enums other than the 2 being fixed by this PR.

Reply:
yeah, it was the original intention to be able to capture all cases found, not just `CheckState` only. The description in the PR could probably do with a better wording to highlight it.